sdk-api-src/content/cfapi/ns-cfapi-cf_placeholder_create_info.md

```diff
@@ -56,10 +56,9 @@ Contains placeholder information for creating new placeholder files or directori
 
 ### -field RelativeFileName
 
-The name of the child placeholder file or directory to be created. 
+The name of the child placeholder file or directory to be created. It should consist only of the file or directory name.
```

Comment:
The name of the placeholder file or directory to be created.

This should be a name, not a path. For example, ...